src/components/Form/Fields/Select.js

```diff
@@ -32,10 +40,18 @@ class Select extends PureComponent {
     );
 
     return (
-      <select className={componentClasses} {...input} {...rest}>
-        {children}
-        {options.map(option => <option value={option}>{option}</option>)}
-      </select>
+      <div className={componentClasses}>
+        { label &&
+          <label
+            htmlFor={this.id}
+            className="form-fields-select__label"
+          >{label}</label>
+        }
+        <select className="form-fields-select__input" {...input} {...rest}>
+          {children}
+          {options.map(option => <option value={option}>{option}</option>)}
```

Comment:
I see a propType warning here when I first come to the screen: `Warning: Each child in an array or iterator should have a unique "key" prop.`

Reply:
👍